concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,251 @@
+
+In other words:
+- We want:
+  - to encourage reporting of security issues (ie: “Please do attack us!”)
+  - to make it easy to report security issues
+  - to maintain and extend our policy of safe defaults / discouraging people
+  from adopting bad security practices
+  - to recognise and show appreciation for the time spent researching and
+  reporting
+  - to make it possible for people who may be at risk to report issues
+  pseudonymously
+
+- We need:
+  - a clear policy, describing both:
+    - how to report; and
+    - what to expect during the process
+  - a main reporting channel:
+    - secondary channels, should the main one fail/not be available/be
+  overloaded.
+  - a list of people who can be reached directly
+  - a place to list security reports and give attribution
+
+
+## Tutorial
+
+This RFC aims to provide a set of documents (wiki pages) that describe the 
+Enarx vulnerability disclosure policy - including the use of embargos - and
+the list of security advisories and fixes.
+
+The two pages are "Vulnerability Reporting and Embargo Policy" and "Enarx 
+Security Advisories."
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.io or on our
+[chat](https://chat.enarx.io) stating you'd like to report a security
+issue, but **without divulging details**.
+
+We treat both email and chat as a public channels, so simply saying that
+you’re trying to get a hold of someone from the security team is fine.
+
+From then, we will establish a more secure area where the issue can be
+discussed, in the form of a 
+[Github Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories).
+
+You will be invited to join this private area to discuss specifics. Doing so 
+allows us to start with a high level of confidentiality and relax it if the 
+issue is less critical, moving to work on the fix in the open.
+
+This methods provides several other advantages :
+1. It avoids email, which is unfortunately very hard dificult to use securely
+2. It makes it possible to privately collaborate to fix the vulnerability in 
+a temporary private fork
+3. It makes it possible to draft a security advisory, and use the draft to 
+privately discuss the impact of the vulnerability on the project.
+4. It makes it possible to easily publish the security advisory and alert the 
+community of the vulnerability
```

Comment:
Suggest the following to reduce wordiness and address some minor grammatical issues:
```suggestion
This method provides several other advantages:
1. Avoids email, which is unfortunately very difficult to use securely
2. Enables private collaboration to fix the vulnerability in 
a temporary private fork
3. Facilitates drafting a security advisory and using the draft to 
privately discuss the impact of the vulnerability on the project
4. Simplifies publishing the security advisory and alerting the 
community of the vulnerability
```

Reply:
+1